inst/extdata/socioeconomics/socioeconomics_ctry.csv

```diff
@@ -1 +1 @@
fsm,"Micronesia, Federated States of",Micronesia,
mda,"Moldova, Republic of",Moldova,Total Former USSR
mco,Monaco,Monaco,
mng,Mongolia,Mongolia,
mne,Montenegro,,
msr,Montserrat,Montserrat,
mar,Morocco,Morocco,
moz,Mozambique,Mozambique,
mmr,Myanmar,Burma,
nam,Namibia,Namibia,
nru,Nauru,Nauru,
npl,Nepal,Nepal,
nld,Netherlands,Netherlands,
ant,Netherlands Antilles,Neth. Antilles,
ncl,New Caledonia,New Caledonia,
nzl,New Zealand,New Zealand,
nic,Nicaragua,Nicaragua,
ner,Niger,Niger,
nga,Nigeria,Nigeria,
niu,Niue,,
nfk,Norfolk Island,,
mnp,Northern Mariana Islands,Northern Mariana Islands,
nor,Norway,Norway,
omn,Oman,Oman,
pci,Pacific Islands Trust Territory,,
pak,Pakistan,Pakistan,
plw,Palau,Palau,
pse,"Palestinian Territory, Occupied",West Bank and Gaza,
pan,Panama,Panama,
png,Papua New Guinea,Papua New Guinea,
pry,Paraguay,Paraguay,
per,Peru,Peru,
phl,Philippines,Philippines,
pcn,Pitcairn Islands,,
pol,Poland,Poland,
prt,Portugal,Portugal,
pri,Puerto Rico,Puerto Rico,
qat,Qatar,Qatar,
reu,Reunion,Reunion,
rom,Romania,Romania,
rus,Russian Federation,Russian Federation,Total Former USSR
rwa,Rwanda,Rwanda,
shn,Saint Helena,Saint Helena,
kna,Saint Kitts and Nevis,St. Kitts Nevis,
lca,Saint Lucia,St. Lucia,
spm,Saint Pierre and Miquelon,St. Pierre and Miquelon,
vct,Saint Vincent and the Grenadines,St. Vincent,
wsm,Samoa,Samoa,
smr,San Marino,San Marino,
stp,Sao Tome and Principe,Sao Tome and Principe,
sau,Saudi Arabia,Saudi Arabia,
sen,Senegal,Senegal,
srb,Serbia,,
scg,Serbia and Montenegro,Serbia/Montenegro,Yugoslavia
syc,Seychelles,Seychelles,
sle,Sierra Leone,Sierra Leone,
sgp,Singapore,Singapore,
svk,Slovakia,Slovakia,Czechoslovakia
svn,Slovenia,Slovenia,Yugoslavia
slb,Solomon Islands,Solomon Islands,
som,Somalia,Somalia,
zaf,South Africa,South Africa,
esp,Spain,Spain,
lka,Sri Lanka,Sri Lanka,
sdn,Sudan,Sudan,
sur,Suriname,Suriname,
sjm,Svalbard and Jan Mayen Islands,,
swz,Swaziland,Swaziland,
swe,Sweden,Sweden,
che,Switzerland,Switzerland,
syr,Syrian Arab Republic,Syria,
twn,Taiwan,Taiwan,
tjk,Tajikistan,Tajikistan,Total Former USSR
tza,"Tanzania, United Republic of",Tanzania,
tha,Thailand,Thailand,
tls,Timor Leste,,
tgo,Togo,Togo,
tkl,Tokelau,,
ton,Tonga,Tonga,
tto,Trinidad and Tobago,Trinidad and Tobago,
tun,Tunisia,Tunisia,
tur,Turkey,Turkey,
tkm,Turkmenistan,Turkmenistan,Total Former USSR
tca,Turks and Caicos,Turks and Caicos Islands,
tuv,Tuvalu,Tuvalu,
uga,Uganda,Uganda,
ukr,Ukraine,Ukraine,Total Former USSR
are,United Arab Emirates,United Arab Emirates,
gbr,United Kingdom,United Kingdom,
usa,United States of America,United States,
ury,Uruguay,Uruguay,
uzb,Uzbekistan,Uzbekistan,Total Former USSR
vut,Vanuatu,Vanuatu,
vat,Vatican,,
ven,Venezuela,Venezuela,
vnm,Viet Nam,Vietnam,
vgb,"Virgin Islands, British",British Virgin Islands,
vir,"Virgin Islands, U.S.",Virgin Islands,
wlf,Wallis and Futuna,Wallis and Futuna,
esh,Western Sahara,Western Sahara,
yem,Yemen,Yemen,
yug,"Yugoslavia, Federal Republic of",,
zmb,Zambia,Zambia,
zwe,Zimbabwe,Zimbabwe,
```

Comment:
Changed source to "Internally generated mapping file."

@d3y419 The ISO codes are not internally generated, but this mapping file--between the Maddison country names (which are not standard ISO country names) to the ISO codes--was internally generated and cannot be accessed anywhere else other than here.